src/docfx/build/toc/parser/TableOfContentsInputModel.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.Docs.Build
 {
     internal class TableOfContentsInputModel
     {
-        public JObject Metadata { get; set; } = new JObject();
```

Comment:
>= new JObject(); [](start = 46, length = 16)

which change this?

Reply:
this is to prevent the output model always has e empty object, it's added by my previous PR

Reply:
+1

---
In reply to: [206783267](https://github.com/dotnet/docfx/pull/3142#discussion_r206783267) [](ancestors = 206783267)

Reply:
Sure. It should align with the ideal output schema. We can discuss it when finalize the schema.

---
In reply to: [206786785](https://github.com/dotnet/docfx/pull/3142#discussion_r206786785) [](ancestors = 206786785)